packages/engineer/src/feature/dev-server.dev-server.env.ts

```diff
@@ -15,27 +15,17 @@ import {
     getExportedEnvironments,
     getResolvedEnvironments,
 } from '@wixc3/engine-scripts';
-import webpackDevMiddleware from 'webpack-dev-middleware';
 import webpack from 'webpack';
 import { WsServerHost } from '@wixc3/engine-core-node';
 import { dirname, resolve } from 'path';
 import { Communication, createDisposables } from '@wixc3/engine-core';
 import { buildFeatureLinks } from '../feature-dependency-graph';
 
-function singleRunWatchFunction(compiler: webpack.Compiler) {
-    // This custom watch optimization only compiles once, but allows us to use webpack dev server
-    // and serve the output from memory
-    return function watch(_: unknown, handler: webpack.ICompiler.Handler) {
-        compiler.run(handler);
-        return {
-            close(cb?: () => void) {
-                if (cb) {
-                    cb();
-                }
-            },
-            invalidate: () => undefined,
-        };
-    };
+// eslint-disable-next-line @typescript-eslint/no-var-requires
```

Comment:
why? are the types wrong when importing?

Reply:
known issue. anything that imports `@types/webpack` is currently broken. there are discussions about this in DefinitelyTyped.